### PR TITLE
docs: optimize hz

### PR DESCRIPTION
**Pull request by chaoranz758**
#### What type of PR is this?

docs

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:optimize hz
zh(optional): 优化 hz 文档

### CODE REVIEW

**Comment by li-jin-gou** [comment]
PATL @FGYFFFF

**Review by li-jin-gou — DISMISSED** [review]

**Comment by li-jin-gou** [comment]
PATL @rogerogers @GuangmingLuo

File: content/en/docs/hertz/tutorials/toolkit/annotation.md [hunk 1]
```diff
@@ -40,6 +43,16 @@ Field annotation tag description can be referenced [supported-tags](https://www.
...
 | api.any             | define ANY methods and routes     |
 
+### hz client
+
+In addition to the annotations provided by [hz](#hz), two additional annotations have been added for client scenarios.
+
+| _Client annotation_ |                         |
+| ------------- | ----------------------- |
+| annotation          | description                    |
+| api.file_name       | specify file     |
```
Comment (FGYFFFF) [inline content/en/docs/hertz/tutorials/toolkit/annotation.md:53]: file_name 这个可以先不用暴露出来。这块 server 端目前没办法处理。

File: content/en/docs/hertz/tutorials/toolkit/annotation.md [hunk 2]
```diff
@@ -83,3 +96,34 @@ service Demo {
...
 }
 ```
+
+### Client annotation
+
+Thrift：
+
+```thrift
+struct Demo {
+    1: string FileValue (api.file_name="file1");
```
Comment (FGYFFFF) [inline content/en/docs/hertz/tutorials/toolkit/annotation.md:106]: 同上

File: content/en/docs/hertz/tutorials/toolkit/command.md [hunk 3]
```diff
@@ -18,16 +18,18 @@ USAGE:
...
 
 VERSION:
-   0.x.x
+   v0.x.x
 
 COMMANDS:
    new      Generate a new Hertz project
    update   Update an existing Hertz project
+   model    Generate model code only
+   client   Generate hertz client based on IDL
```
Comment (FGYFFFF) [inline content/en/docs/hertz/tutorials/toolkit/command.md:27]: hertz -> Hertz ; Hertz 这块首字母最好统一大写

File: content/en/docs/hertz/tutorials/toolkit/annotation.md (hunk 2 above)
Comment (li-jin-gou) [reply content/en/docs/hertz/tutorials/toolkit/annotation.md]: done

File: content/en/docs/hertz/tutorials/toolkit/annotation.md (hunk 1 above)
Comment (li-jin-gou) [reply content/en/docs/hertz/tutorials/toolkit/annotation.md]: done

File: content/en/docs/hertz/tutorials/toolkit/command.md (hunk 3 above)
Comment (li-jin-gou) [reply content/en/docs/hertz/tutorials/toolkit/command.md:27]: 这里需要先更新 hz

**Comment by li-jin-gou** [comment]
暂时先合进去了 @chaoranz758 @FGYFFFF @GuangmingLuo @rogerogers

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**